### PR TITLE
add sanitizer

### LINKED ISSUE #3832: [Ops] Research integrating Snyk into CI/CD process
We now have Snyk integrated with our repository and it's reporting issues. 

### This task is done when...

- [ ] Snyk issues are addressed
- [ ] Snyk validation has replaced npm audit

### PR DESCRIPTION
**Pull request by thetif**
Resolves #3832 

**Description-**
Resolve issues discovered by the snyk plugin in vscode.

**This pull request changes...**

- res.send to res.json where applicable
- adds csrf, cookie-parser, and express-validator to improve security
- adds some validation to param/body inputs from users
- 

**This pull request also touches…**

- updated the mongo settings to match new mongo configs

**This pull request was tested in the follow ways…**

- automated tests
- manual verification that site is still functional
### This pull request is ready to review when...

- [x] Automated tests are updated (and all tests are passing)
- [ ] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented

### This pull request can be merged when…

- [ ] Code has been reviewed by someone other than the original author
- [ ] QA has verified the accessibility and functionality related to the change
- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  ],
+  "license": "CC0-1.0",
+  "bugs": {
+    "url": "https://github.com/CMSgov/eAPD/issues"
+  },
+  "homepage": "https://github.com/CMSgov/eAPD#readme",
+  "engines": {
+    "node": "16.13.2"
+  }
+}
```
Comment (tbolt) [inline package.json:56]: just curious, whats the intention with adding this top-level?

**Review by tbolt — APPROVED** [review]
looks good 👍

**Review by mirano-darren — APPROVED** [review]
Looks good! Good stuff, Everything looks unphased and booted up as normal.
Quick note: If you get a Mongo error on "docker compose up" you need to update your env_vars

**Review by jeromeleecms — APPROVED** [review]
Fixed the deployment issue with Mongo too!

**Pull request merged**